library/common/internal/text_input_model.h

```diff
@@ -82,13 +82,24 @@ class TextInputModel {
   // Returns the state in the form of a platform message.
   Json::Value GetState() const;
 
+  // Id of the text input client.
   int client_id() const { return client_id_; }
 
+  // Keyboard type of the client. See available options:
+  // https://github.com/flutter/flutter/blob/0468742910c6e9230533e8e2c616520116b906e6/packages/flutter/lib/src/services/text_input.dart#L25
```

Comment:
Since the file this is linking to will become increasingly stale over time, maybe it would be better to say something like "See text_input.dart in packages/flutter".

Similar below.